bundles/org.openhab.binding.modbus.helioseasycontrols/README.md

```diff
@@ -0,0 +1,306 @@
+| humidityControlSetValue          | humidityControl | Humidity control set value (in percent)                          | Number:Dimensionless     | %    | 20 - 80                                                                                              |
+| humidityControlSteps             | humidityControl | Humidity control steps (in percent)                              | Number:Dimensionless     | %    | 5 - 20                                                                                               |
+| humidityStopTime                 | humidityControl | Humidity stop time in hours (0-24)                               | Number:Time              | h    | 0 - 24                                                                                               |
+| co2ControlStatus                 | co2Control      | CO2 control status                                               | Number                   |      | 0 = off, 1 = stepped, 2 = stepless                                                                   |
+| co2ControlSetValue               | co2Control      | CO2 control set value (in ppm)                                   | Number:Dimensionless     | ppm  | 300 - 2000                                                                                           |
+| co2ControlSteps                  | co2Control      | CO2 control steps (in ppm)                                       | Number:Dimensionless     | ppm  | 50 - 400                                                                                             |
+| vocControlStatus                 | vocControl      | VOC control status                                               | Number                   |      | 0 = off, 1 = stepped, 2 = stepless                                                                   |
+| vocControlSetValue               | vocControl      | VOC control set value (in ppm)                                   | Number:Dimensionless     | ppm  | 300 - 2000                                                                                           |
+| vocControlSteps                  | vocControl      | VOC control steps (in ppm)                                       | Number:Dimensionless     | ppm  | 50 - 400                                                                                             |
+| comfortTemp                      | unitConfig      | Comfort Temperature                                              | Number:Temperature       | °C   | 10.0 - 25.0                                                                                          |
+| timeZoneDifferenceToGmt          | general         | Time Zone Difference to GMT                                      | Number:Time              | h    | -12 - 14                                                                                             |

```

Comment:
What's the purpose of this Channel? You can retrieve the UTC offset via OH. See https://www.openhab.org/javadoc/v2.5/org/eclipse/smarthome/core/i18n/timezoneprovider

Reply:
That's a the way to tell the device which time zone difference to use. That's how it's defined in the device's Modbus protocol specification.

Reply:
I.e. you need this info for calculating the correct `DateTime` in the binding. There should be no need to expose it to the OH user, then.